src/utils/git.cpp

```diff
@@ -79,7 +79,25 @@ void Utils::Git::pushCurrentNoteFolder() {
     auto* process = new QProcess();
     process->setWorkingDirectory(NoteFolder::currentLocalPath());
 
+    if (!executeGitCommand(git, QStringList{"pull", "--no-commit", "--no-ff"}, process)) {
```

Comment:
What if the error is "git command not found"?

Reply:
Since the commit is done before, there would be an error message earlier.